Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSock.yml

```
@@ -0,0 +1,438 @@
+    - ClothMade
+    - WhitelistChameleon
+
+- type: entity
+  parent: ClothingShoesBase
+  id: ClothingUnderSocksStripedCargoQm
+  name: Logistics striped socks
+  description: Striped Socks that make you feel like becoming an independant Station State
+  components:
+  - type: Sprite
+    sprite: DeltaV/Clothing/Under/Socks/ClothingUnderSocksStripedCargoQm.rsi
+  - type: Clothing
+    sprite: DeltaV/Clothing/Under/Socks/ClothingUnderSocksStripedCargoQm.rsi
+  - type: Food
+    requiresSpecialDigestion: true
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 30
+        reagents:
+        - ReagentId: Fiber
+          Quantity: 30
+  - type: Tag
+    tags:
+    - ClothMade
+    - WhitelistChameleon
+
+- type: entity
+  parent: ClothingShoesBase
+  id: ClothingUnderSocksStripedEngineeringAtmos
+  name: Atmos Tech striped socks
+  description: Striped socks to make you stand out in an emergency
+  components:
+  - type: Sprite
+    sprite: DeltaV/Clothing/Under/Socks/ClothingUnderSocksStripedEngineeringAtmos.rsi
+  - type: Clothing
+    sprite: DeltaV/Clothing/Under/Socks/ClothingUnderSocksStripedEngineeringAtmos.rsi
+  - type: Food
+    requiresSpecialDigestion: true
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 30
+        reagents:
+        - ReagentId: Fiber
+          Quantity: 30
+  - type: Tag
+    tags:
+    - ClothMade
+    - WhitelistChameleon
+
+- type: entity
+  parent: ClothingShoesBase
+  id: ClothingUnderSocksStripedEngineering
+  name: Engineer striped socks
+  description: Striped socks to make you stand out in an emergency
+  components:
+  - type: Sprite
+    sprite: DeltaV/Clothing/Under/Socks/ClothingUnderSocksStripedEngineering.rsi
+  - type: Clothing
+    sprite: DeltaV/Clothing/Under/Socks/ClothingUnderSocksStripedEngineering.rsi
+  - type: Food
+    requiresSpecialDigestion: true
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 30
+        reagents:
+        - ReagentId: Fiber
+          Quantity: 30
+  - type: Tag
+    tags:
+    - ClothMade
+    - WhitelistChameleon
+
+- type: entity
+  parent: ClothingShoesBase
+  id: ClothingUnderSocksStripedScience
+  name: Science striped socks
+  description: These Striped socks make you want to do some Science
+  components:
+  - type: Sprite
+    sprite: DeltaV/Clothing/Under/Socks/ClothingUnderSocksStripedScience.rsi
+  - type: Clothing
+    sprite: DeltaV/Clothing/Under/Socks/ClothingUnderSocksStripedScience.rsi
+  - type: Food
+    requiresSpecialDigestion: true
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 30
+        reagents:
+        - ReagentId: Fiber
+          Quantity: 30
+  - type: Tag
+    tags:
+    - ClothMade
+    - WhitelistChameleon
+
+- type: entity
+  parent: ClothingShoesBase
+  id: ClothingUnderSocksStripedSecurity
+  name: Security striped socks
+  description: These striped socks make you feel like beating up a mime
```

```suggestion
  name: striped socks
  description: These striped socks make you feel like beating up an evildoer.
```
Trying not to allude to brutality being okay.